types/bbob__parser/utils.d.ts

```diff
@@ -0,0 +1,41 @@
+import { Node } from "@bbob/plugin-helper";
+
+export interface Options {
+    onSkip: boolean;
+}
+
+export class CharGrabber {
```

Comment:
IMO you should not do this. The utils export factory methods that wrap CharGrabber instance creation. No classes are exported IMO in original module. Please export an interface or make this abstract, just to prevent users of this DT package from creating instance of CharGrabber class directly, same for 'NodeList'.
Am I seeing this right? 
(we are trying to align export TS types with actual features of described JS modules)

Reply:
good catch - I just made them interfaces since the constructor is never exported